### ISSUE TITLE
Suggest: no use of attach in examples

### ISSUE THREAD

**Issue by richelbilderbeek** [leeshawn/SKAT#10]
Dear SKAT authors, thanks for writing this package! I do have a minor suggestion and I do volunteer to submit a Pull Request , which I describe below.

When viewing the SKAT::SKAT doc ...

```R
library(SKAT)
?SKAT
```

we find this example code:

```
data(SKAT.example)
attach(SKAT.example)
# [...]
obj<-SKAT_Null_Model(y.c ~ 1, out_type="C")
```

I suggest to avoid the use of attach in this example, as lintr, the Tidyverse coding standard, states that [attach is an undesirable function](https://github.com/jimhester/lintr/blob/master/R/zzz.R#L147).

The easy solution would be to write the same code as such:

```
data(SKAT.example)
# [...]
obj <- SKAT_Null_Model(data = SKAT.example, formula = y.c ~ 1, out_type = "C")
```

In this way, the examples depend less on the state of `env`.

Could you agree that following the most used R coding standard is a good idea? Again, I do volunteer to do so :-)

**Comment by richelbilderbeek** [comment]
Contacted the SKAT maintainer by email:

Dear SKAT maintainer,

I contact you as I have posted two Issues at the SKAT GitHub repo and am unsure if you have read these.

If you have and just haven't had the time yet, I would understand :-)

Looking forward to a response one day and cheers, Richel Bilderbeek

 * https://github.com/leeshawn/SKAT/issues/10

 * https://github.com/leeshawn/SKAT/issues/11

**Comment by leeshawn** [comment]
Thanks for the suggestion. I modified the example

**Comment by leeshawn** [comment]
BTW, adding column name isn't done yet. It will be great if you can do this.

**Comment by richelbilderbeek** [comment]
Fixed with 4d12a26334bd628f438d92bd554aa5905a747517, thanks @leeshawn !
